### PR TITLE
Enterprise awareness docs

### PR DESCRIPTION
**Pull request by bmckercher123**
Added 4 new docs to /Security/Calico Enterprise

** WAIT until I remove this message to review **

- Merge to /latest 
- Approved by Andy and Alex
- Copy edit only
- This is marketing content so may not follow tech doc standards

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 3299ac7871cf812bf68ec2a0b2494065b5c83ea4

https://deploy-preview-3207--calico.netlify.com

File: security/calico-enterprise/compliance.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+Calico Enterprise flow logs contain important Kubernetes metadata including (but not limited to):
+
+- Source and destination namespaces
+- Source and destination pods
+- Labels for source and destination pods
+- Originating IP address for ingress connections
+- Which policies evaluated the connection and whether the policy allowed or denied the connection
+
+With Calico Enterprise flow logs, can you prove to security and auditors that your policies are in place and being enforced.
```
Comment (lmm) [inline security/calico-enterprise/compliance.md]: `can you` => `you can`

File: security/calico-enterprise/compliance.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+### Configuration auditing
+
+Your security team or auditor may also ask about the configuration of your Kubernetes cluster to ensure it aligns with best-practices for security. The industry standard spec for Kubernetes security configuration is the [Center for Internet Security Benchmark for Kubernetes](https://www.cisecurity.org/benchmark/kubernetes/) (CIS).
+
+Calico Enterprise audits the configuration of your cluster and reports on compliance and noncompliance for CIS Level 1 and Level 2 security. Configuration audits are periodically executed, enabling a historical view of the configuration over time. 
+
+![configuration-compliance]({{site.baseurl}}/images/configuration-compliance.png)
+
+The specific findings can be downloaded as a CSV file that provides detailed information about the current value of each configuration setting and the suggested configuration per the CIS standard. If the fix is not viable but is an acceptable risk to take within your organization, you can configure the report specification to exclude that test index so that it no longer shown in the dashboard.
```
Comment (lmm) [inline security/calico-enterprise/compliance.md]: What is a `test index`?

File: security/calico-enterprise/egress-access-controls.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 calico_enterprise: true
 ---
 
 ### Egress access control to external endpoints
 
-Fine-grained access control between a Pod and external service outside the cluster generally requires a firewall rule or equivalent between your Pod and the services you need to connect to. Since traditional firewall rules are defined using static IP addresses, the dynamic nature of Pod IP assignment creates challenges defining firewall rules. Kubernetes/Calico network policies are designed to abstract away from IP addresses in favor of label selectors, but still require external services outside of the cluster to be identified by IP address.
+Fine-grained access control between a pod and external service outside the cluster generally requires a firewall rule or equivalent between your pod and the services you need to connect to. Since traditional firewall rules are defined using static IP addresses, the dynamic nature of pod IP assignment creates challenges defining firewall rules. Kubernetes/Calico network policies are designed to abstract away from IP addresses in favor of label selectors, but still require external services outside of the cluster to be identified by IP address.
```
Comment (lmm) [inline security/calico-enterprise/egress-access-controls.md]: `creates challenges defining firewall rules` => `creates challenges in defining firewall rules` ?

File: security/calico-enterprise/compliance.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+- Policy tiers
+- Flow logs
+- Audit logs
+- Configuration auditing
+- Compliance reports
+
+### Policy tiers
+
+Policy tiers are ordered collections of Calico policies. Policy tiers enable privileged users to define security policies that take precedence over other user’s policies. A couple common use cases include:
```
[the remark this answers is not in the document]
Comment (bmckercher123) [reply security/calico-enterprise/compliance.md]: either "a couple of" or "a few common"

[the remark this answers is not in the document]
Comment (bmckercher123) [reply security/calico-enterprise/compliance.md]: FIXED

File: security/calico-enterprise/compliance.md (hunk 1 above)
Comment (bmckercher123) [reply security/calico-enterprise/compliance.md]: Fixed

File: security/calico-enterprise/compliance.md (hunk 2 above)
Comment (bmckercher123) [reply security/calico-enterprise/compliance.md]: If you look at screen shot, there is a column called "Index"

File: security/calico-enterprise/egress-access-controls.md (hunk 3 above)
Comment (bmckercher123) [reply security/calico-enterprise/egress-access-controls.md]: Fixed.

**Pull request merged**